components/mdx/GridUsedBy.tsx

```diff
@@ -1,12 +1,13 @@
-const Image = (props) => {
+const Image = ({ size = 'contain', url, ...rest }) => {
   return (
     <a
-      style={{
-        backgroundImage: ` url(${props.url})`,
-        backgroundSize: props.size || 'contain',
-        backgroundRepeat: 'no-repeat',
-        backgroundPosition: ' center center',
-      }}
+      {...rest}
+      css={`
+        background-image: ' url(${url})';
```

Comment:
Is this valid CSS? I think we'll have to remove the single quotes here.

Reply:
It did not, fixed